tests/xlsx.test.js

```diff
@@ -79,4 +84,12 @@ describe('are xlsx slots present in schema', () => {
 
         expect(extraSlots).toStrictEqual([]);
     });
+
+    test.skip('ap', () => {
+        const xlsxSlots = getXlsxSlots('report_ap.xlsx');
+        const schemaSlots = getSchemaSlots('ap.json');
+        const extraSlots = [...xlsxSlots].filter((x) => !schemaSlots.has(x));
+
+        expect(extraSlots).toStrictEqual([]);
+    });
```

Comment:
We had temporarily skipped this test for Chennai because we didn't have the right schema, is this test not passing for AP? Can you replace `test.skip()` with simply `test()` and run it?